lib/Agrammon/DB/Dataset.pm6

```diff
@@ -373,17 +389,29 @@ class Agrammon::DB::Dataset does Agrammon::DB {
                          !! self!delete-variable($var);
     }
 
-    method rename-instance($old-instance, $new-instance, $pattern) {
+    method rename-instance($old-name, $new-name, $pattern) {
```

Comment:
Minor style point (but it can be done as a PR across the codebase if you agree): when something isn't going to return a useful value, I often add a `--> Nil` at the end of the signature to document/enforce that (enforce as in it means no values leak out and accidentally get used as if they were intended return values).